lib/main/widgets/pet_list_item.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class PetListItem extends StatelessWidget {
+  final String namePet;
+  final String breedPet;
+
+  //final Image imagePet;
```

Comment:
I still want you to add the image from url functionality... 
Why did you resolve previous suggestion if you did nothing?